frontend/packages/kubevirt-plugin/src/components/create-vm/create-vm.tsx

```diff
@@ -200,6 +202,22 @@ export const CreateVM: React.FC<RouteComponentProps<{ ns: string }>> = ({ match,
     restoreDefaultSSHSettings,
   } = useSSHKeys();
   const { createOrDeleteSSHService } = useSSHService();
+  const [storageClasses, storageClassLoaded] = useK8sWatchResource<K8sResourceCommon[]>({
+    kind: StorageClassModel.kind,
+    isList: true,
+  });
+  const [storageClassDefaultName, setStorageClassDefaultName] = React.useState<string>();
+
+  React.useEffect(() => {
+    if (storageClasses && storageClassLoaded) {
+      const storageClassDefault = storageClasses?.find(
+        (sc) => sc?.metadata?.annotations[DEFAULT_SC_ANNOTATION] === 'true',
+      );
+      storageClassDefault && setStorageClassDefaultName(storageClassDefault?.metadata?.name);
+    }
+  }, [storageClasses, storageClassLoaded]);
```

Comment:
Convert these two hooks into one useMemo :)

Reply:
Cool let me is this works. If so I'll push changes